samples/SocialSample/Startup.cs

```diff
@@ -128,44 +137,32 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerfactory)
                 }
             });
 
-            // You must first create an app with live.com and add it's ID and Secret to your config.json or user-secrets.
-            /* https://account.live.com/developers/applications
-            The MicrosoftAccount service has restrictions that prevent the use of http://localhost:54540/ for test applications.
-            As such, here is how to change this sample to uses http://mssecsample.localhost.this:54540/ instead.
-
-            Edit the hosting.json file and add "server.urls": "http://mssecsample.localhost.this:54540/".
-
-            From an admin command console first enter:
-             notepad C:\Windows\System32\drivers\etc\hosts
-            and add this to the file, save, and exit (and reboot?):
-             127.0.0.1 MsSecSample.localhost.this
-
-            [WebListener] Then you can choose to run the app as admin (see below) or add the following ACL as admin:
-             netsh http add urlacl url=http://mssecsample.localhost.this:54540/ user=[domain\user]
-
-            The sample app can then be run via:
-             dnx web
+            /* Azure AD app model v2 has restrictions that prevent the use of plain HTTP for redirect URLs.
+               Therefore, to authenticate through microsoft accounts, tryout the sample using the following URL:
+               https://localhost:54541/
             */
+
+            // See config.json
+            // https://apps.dev.microsoft.com/
             app.UseOAuthAuthentication(new OAuthOptions
             {
                 AuthenticationScheme = "Microsoft-AccessToken",
-                DisplayName = "MicrosoftAccount-AccessToken - Requires project changes",
+                DisplayName = "MicrosoftAccount-AccessToken",
                 ClientId = Configuration["msa:clientid"],
                 ClientSecret = Configuration["msa:clientsecret"],
                 CallbackPath = new PathString("/signin-microsoft-token"),
                 AuthorizationEndpoint = MicrosoftAccountDefaults.AuthorizationEndpoint,
                 TokenEndpoint = MicrosoftAccountDefaults.TokenEndpoint,
-                Scope = { "wl.basic" },
+                Scope = { "openid", "https://graph.microsoft.com/user.read" },
```

Comment:
Yup this has nothing to do with openid stuff. `user.read` is quite enough.
The same applies to https://github.com/aspnet/Security/blob/dev/samples/SocialSample/Startup.cs#L92 ?